app/views/users/_sidebar.html.erb

```diff
@@ -48,34 +48,21 @@
       <div class="widget">
         <div class="daily-stats">
           <h4>Daily Stats</h4>
-          <p>3 Visits</p>
-          <p>35 Minutes on Pages</p>
-          <p>4 Articles Read</p>
-          <p>1456 Words Read</p>
+          <p id="visits-per-day"><%= pluralize @user.number_visits_per_day, "Visit" %> </p>
+          <p id="average-active-time-per-day"><%= pluralize @user.average_active_time_per_day, "Second" %>Spent Reading</p>
+          <p id="average-number-articles-per-day"><%= pluralize @user.number_articles_per_day, "Article" %> Read</p>
+          <p id="average-number-words-per-day"><%= pluralize @user.number_words_per_day, "Word" %> Read </p>
         </div>
         </div>
         <div class="sidebar-data">
         <h4>Overall Stats</h4>
-          <p>
-            <%= pluralize @user.articles.published.size, "Post" %> Published
-          </p>
-          <p>
-            <%= pluralize @user.comments.where(deleted: false).size, "Comment" %> Written
-          </p>
-          <p>
-            <%= pluralize @user.decorate.cached_followed_tags.size, "Tag" %> Followed
-          </p>
-          <p>10 Total Visits</p>
-          <p>34 Articles Read</p>
-          <p>5 Comments Received</p>
-          <p>18992 Words Read</p>
-        </div>
-        <div class="daily-stats">
-          <h3>Daily Stats</h3>
-          <p>3 Visits</p>
-          <p>35 Minutes on Pages</p>
-          <p>4 Articles Read</p>
-          <p>1456 Words Read</p>
+          <p> <%= pluralize @user.articles.published.size, "Post" %> Published </p>
+          <p> <%= pluralize @user.comments.where(deleted: false).size, "Comment" %> Written </p>
+          <p> <%= pluralize @user.decorate.cached_followed_tags.size, "Tag" %> Followed </p>
+          <p id="number-visits"> <%= pluralize @user.sign_in_count, "Visit" %> </p>
+          <p id="number-articles-read"> <%= pluralize @user.number_articles_read, "Article" %> Read </p>
+          <p> 5 Comments Received</p>
```

Comment:
this one still needs to be added

Reply:
Noted. I think the stats you have so far are enough for the MVP.